src/modrover/strategies/base.py

```diff
@@ -0,0 +1,77 @@
+from abc import ABC, abstractmethod
+from typing import Dict, Iterable, Set
+
+from modrover.learnerid import LearnerID
+from modrover.learner import Learner
```

Comment:
Just curious if it's best practice to have each class in a separate module? Seems like Learner and LearnerID could be in the same module, but maybe it's a preference thing?

Reply:
I like `LearnerID` and `Learner` in one module :)